en/installation/web-and-post-installation.md

```diff
@@ -20,36 +20,36 @@ Then click on **Next**.
 
 ![image](../assets/installation/acentreoncheckmodules.png)
 
-Define pathes used by monitoring engine. We recommand to use defaults.
+Define paths used by monitoring engine. We recommend to use defaults.
 
 Then click on **Next**.
 
 ![image](../assets/installation/amonitoringengine2.png)
 
-Define pathes used by broker module. We recommand to use defaults.
+Define paths used by broker module. We recommend to use defaults.
 
 Then click on **Next**.
 
 ![image](../assets/installation/abrokerinfo2.png)
 
-Define informations for the admin account creation.
+Define the information needed for the admin account creation.
 
 Then click on **Next**.
 
 ![image](../assets/installation/aadmininfo.png)
 
-Provide informations to connect to the database instance.
+Provide the information needed to connect to the database instance.
 
 By default, the instance address is set to *localhost*, the instance root
-user is set to *root* and the root password is empty.
+user is set to *root* and the root password is empty. Since MariaDB 10.5, it has been mandatory to add a password for the root user as indicated in [Secure the DBMS installation] (../administration/secure-platform.html#securing-the-installation-of-the-dbms) and the various installation methods. The password configured in the mysql_secure_installation method will be used for the root password. 
```

Comment:
```suggestion
user is set to *root* and the root password is empty. Since MariaDB 10.5, it has been mandatory to add a password for the root user as indicated in [Secure the DBMS installation](../administration/secure-platform.html#securing-the-installation-of-the-dbms) and the various installation methods. The password configured in the mysql_secure_installation method will be used for the root password. 
```